pkg/rosa/idps.go

```diff
@@ -14,7 +13,7 @@ const (
 
 // CreateAdminUserIfNotExist creates a new admin user withe username/password in the cluster if username doesn't already exist.
 // the user is granted admin privileges by being added to a special IDP called `cluster-admin` which will be created if it doesn't already exist.
-func CreateAdminUserIfNotExist(client *ocm.Client, clusterID, username, password string) error {
```

Comment:
We are using the github.com/openshift/rosa/pkg/ocm in order to avoid duplicate the effort of creating a lib that communicate with OCM. Please keep using it for this first integration test iteration

Reply:
Same as above

Reply:
okay, just github duplication